### ISSUE TITLE
List and Grid View in Questions

### ISSUE THREAD

**Issue by avsingh999** [publiclab/plots2#3407]
### Please describe the problem (or idea)
In  https://publiclab.org/questions/ page if we select list view then All questions show in lists but if  i refresh the page or move to another page then It's changing to again  Grid view
> What happened just before the problem occurred? Or what problem could this idea solve?

> What did you expect to see that you didn't?
If we select view that should be fixed every time untill unless user change that view

### Please show us where to look

https://publiclab.org/questions/

### What's your PublicLab.org username?
999_avsingh999
> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by namangupta01** [comment]
Nice issue @avsingh999 , i think we can do this by using localstorage.

**Comment by Paarmita** [comment]
@avsingh999 grid view is the default, so when we refresh or move to a new page, local storage must be in its default view. Correct me if I'm wrong?

**Comment by jywarren** [comment]
Hi! I think unfortunately that we'll be removing this switch feature soon
because the list view is not very compact. But if you'd like to modify it
to use a table view instead of a grid view, that could work!

A related and very high priority issue we've been trying to solve for 2
years is this one, if you want to give it a try!

https://github.com/publiclab/plots2/issues/1097

Thanks for your help!!!

**Comment by Paarmita** [comment]
@jywarren Yes list view has a lot of white spaces. Can u plz specify how table view would look like?
And does these view changes are through local storage?
